cluster-autoscaler/simulator/drain_test.go

```diff
@@ -249,8 +191,153 @@ func TestGetPodsToMove(t *testing.T) {
 		},
 	}
 
-	r10SkipPodsThatShouldBeTerminatedTrue, _, blockingPod, err := GetPodsToMove(schedulerframework.NewNodeInfo(pod10, pod10Terminated, pod10Terminating), deleteOptions, nil, nil, testTime)
-	assert.NoError(t, err)
-	assert.Nil(t, blockingPod)
-	assert.ElementsMatch(t, []*apiv1.Pod{pod10, pod10Terminating}, r10SkipPodsThatShouldBeTerminatedTrue)
+	testCases := []struct {
+		desc         string
+		pods         []*apiv1.Pod
+		pdbs         []*policyv1.PodDisruptionBudget
+		filters      []drainability.Filter
+		wantPods     []*apiv1.Pod
+		wantDs       []*apiv1.Pod
+		wantBlocking *drain.BlockingPod
+		wantErr      bool
+	}{
+		{
+			desc:    "Unreplicated pod",
+			pods:    []*apiv1.Pod{pod1},
+			wantErr: true,
+			wantBlocking: &drain.BlockingPod{
+				Pod:    pod1,
```

Comment:
First of all, thanks for refactoring the unittest, much appreciated :)

Know putting my nitpicky hat on - maybe the test would be more readable if we renamed the pods to have more descriptive names? ex. rsPod, dsPod, localStoragePod?

Reply:
Yeah, I definitely should've done that while refactoring, but was just too lazy :) Will update in a moment.